Shared/Theme/Themeable.swift

```diff
@@ -21,11 +22,19 @@ extension Themeable {
         themeObserver = notificationCenter.addObserver(name: .ThemeDidChange,
                                                        queue: mainQueue) { [weak self] _ in
             self?.applyTheme()
-            self?.updateThemeApplicableSubviews()
         }
     }
 
-    public func updateThemeApplicableSubviews() {
+    public func listenForThemeChange(_ subview: UIView) {
```

Comment:
Protocol states `func listenForThemeChange(_ view: UIView)`, parameter name should align. 
```suggestion
    public func listenForThemeChange(_ view: UIView) {
```

Reply:
oops, I was doing last minute refactoring and made it subview but didn't change the protocol